Sources/Quick/NSBundle+CurrentTestBundle.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension NSBundle {
+
+    /**
+     Locates the first bundle with a '.xctest' file extension.
```

Comment:
When I looked into that question recently I discovered that Xcode launches the entire test host from scratch for each test bundle in the scheme, so I don't know of any situation where multiple test bundles would be loaded into a single process.